metadata-ingestion/src/datahub/ingestion/source/sql/mssql/sql_queries/stored_procedures_dependencies_query.sql

```diff
@@ -0,0 +1,253 @@
+BEGIN;
+IF OBJECT_ID('tempdb.dbo.#ProceduresDependencies', 'U') IS NOT NULL
+    DROP TABLE #ProceduresDependencies;
+
+CREATE TABLE #ProceduresDependencies(
```

Comment:
will we need any new permissions for this? if so, we need to update the documentation